test-cluster.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+#set -x
+
+region=us-west-1
```

Comment:
Not a big deal, but it looks like John was using us-west-2 to generate his images, so something to consider.

Reply:
This is for test and can be changed when  calling the script. This is just an initial/default value. 

./test-cluster.sh --region=${REGION} pass whatever region we want 

or using makefile 

 make -C cloud/aws/amazon-ecs-ami REGION=us-west-1 test-al2

Reply:
Note: region is also a parameter now when building AMI

Reply:
Got it.  That's cool that we can override.  Very useful to test AMIs in different regions.